src/request_items.rs

```diff
@@ -210,10 +214,17 @@ impl RequestItems {
                 RequestItem::JsonField(key, value) => {
                     body.insert(key, value);
                 }
+                RequestItem::JsonFieldFromFile(key, value)
+                | RequestItem::DataFieldFromFile(key, value) => {
+                    let mut file = File::open(value)?;
+                    let mut contents = String::new();
+                    file.read_to_string(&mut contents)?;
```

Comment:
You can use [`std::fs::read_to_string`](https://doc.rust-lang.org/std/fs/fn.read_to_string.html) (here and elsewhere)